tests/pkg_libfixmath/do-test.py

```diff
@@ -78,12 +78,16 @@ def main():
             abs_error = abs(res_locals['result'] - float(res_locals['expected']))
             res_locals['result'] = '{:.4f}'.format(res_locals['result'])
             if abs_error > ABS_ERROR_LIMIT:
-                print('{}: {} != {}, {:.4f} > {}'.format(res_locals['input'], res_locals['result'], res_locals['expected'],
-                                                         abs_error, ABS_ERROR_LIMIT))
+                print('{}: {} != {}, {:.4f} > {}'.format(
+                    res_locals['input'],
+                    res_locals['result'],
+                    res_locals['expected'],
+                    abs_error,
+                    ABS_ERROR_LIMIT))
                 errors += 1
-        except:
+        except Exception as exc:
             errors += 1
-            print('ERROR {}'.format(line))
+            print('ERROR: {}: {}'.format(line, exc))
```

Comment:
The commit message states "tests/pkg_libfixmath: fix PEP8 issues". Its unclear to me which PEP8 issue is fixed by this line (comment originally made in https://github.com/RIOT-OS/RIOT/pull/12706#discussion_r346250974).

Reply:
The issue was: E722 "bare except". So the fix is to use:
```py
    except Exception as exc:
```
And make use of it in the error message. In the end, it improves the message: you know it's an error, you get the line and you get the complete exception message.